### PR TITLE
feat(internal): Add 'rollout' flag to deploy

### PR DESCRIPTION
**Pull request by craciunoiuc**
### Prerequisite checklist

  - [ ] Read the [contribution guidelines](https://unikraft.org/docs/contributing/) regarding submitting new changes to the project;
  - [ ] Tested your changes against relevant architectures and platforms;
  - [ ] Ran `make fmt` on your commit series before opening this PR;
  - [ ] Updated relevant documentation.

### Description of changes

This is a quick fix workaround to #1187

### CODE REVIEW

File: internal/cli/kraft/cloud/deploy/deploy.go [hunk 1]
```diff
@@ -218,6 +225,69 @@ func (opts *DeployOptions) Run(ctx context.Context, args []string) error {
...
+
+					if len(oldInsts) != 1 {
+						return fmt.Errorf("expected 1 instance, got %d", len(oldInsts))
+					}
+
+					if _, err := instanceClient.StopByUUIDs(ctx, int(time.Minute.Milliseconds()), oldInsts[0].UUID); err != nil {
+						return fmt.Errorf("could not stop the old instance: %w", err)
+					}
+
+					log.G(ctx).Infof("waiting one minute for the old instance to stop")
```
Comment (antoineco) [inline internal/cli/kraft/cloud/deploy/deploy.go]: `s/Infof/Info`

File: internal/cli/kraft/cloud/deploy/deploy.go [hunk 2]
```diff
@@ -218,6 +225,69 @@ func (opts *DeployOptions) Run(ctx context.Context, args []string) error {
...
+					instanceClient := kraftcloud.NewInstancesClient(
+						kraftcloud.WithToken(config.GetKraftCloudTokenAuthConfig(*opts.Auth)),
+						kraftcloud.WithDefaultMetro(opts.Metro),
+					)
+
+					var oldInsts []instances.GetResponseItem
+					if utils.IsUUID(opts.Rollout) {
+						oldInsts, err = instanceClient.GetByUUIDs(ctx, opts.Rollout)
+						if err != nil {
+							return fmt.Errorf("could not retrieve old instance, invalid UUID: %w", err)
```
Comment (antoineco) [inline internal/cli/kraft/cloud/deploy/deploy.go]: The error here won't necessarily be related to an "invalid UUID".

**Review by antoineco — APPROVED** [review]
Approving with 2 minor comments

File: internal/cli/kraft/cloud/deploy/deploy.go (hunk 2 above)
Comment (craciunoiuc) [reply internal/cli/kraft/cloud/deploy/deploy.go]: makes sense, will edit

**Pull request merged**